test/data/manifests/fedora-coreos-container.mpp.yaml

```diff
@@ -0,0 +1,263 @@
+      - id: BIOS-BOOT
+        size: 2048
+        type: 21686148-6449-6E6F-744E-656564454649
+        bootable: true
+        uuid: FAC7F1FB-3E8D-4137-A512-961DE09A5549
+      - id: EFI-SYSTEM
+        size: 260096
+        type: C12A7328-F81F-11D2-BA4B-00A0C93EC93B
+        uuid: 68B2905B-DF3E-4FB3-80FA-49D1E773AA33
+      - id: boot
+        size: 786432
+        type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        uuid: 61B2905B-DF3E-4FB3-80FA-49D1E773AA32
+      - id: root
+        # XXX: Dynamically set this size in the future
+        size: 8388608
+        type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        uuid: CA7D7CCB-63ED-4C53-861C-1742536059CC
+# sources:
+#   org.osbuild.ostree:
+#     items:
+#       2c39e9b6a6b26226b626f4daf815d5307a96b2cd57463e94f7c618e312a61980:
+#         remote:
+#           url: https://kojipkgs.fedoraproject.org/ostree/repo/
+pipelines:
+  - mpp-import-pipelines:
+      path: fedora-vars.ipp.yaml
+  - mpp-import-pipeline:
+      path: fedora-build-v2.ipp.yaml
+      id: build
+    runner:
+      mpp-format-string: org.osbuild.fedora{release}
+  - name: image-tree
+    # build: name:build
+    source-epoch: 1659397331
+    stages:
+      - type: org.osbuild.ostree.init-fs
+      - type: org.osbuild.ostree.os-init
+        options:
+          osname: fedora-coreos
+      - type: org.osbuild.ostree.config
+        options:
+          repo: /ostree/repo
+          config:
+            sysroot:
+              readonly: false
+              bootloader: none
+      - type: org.osbuild.mkdir
+        options:
+          paths:
+            - path: /boot/efi
+              mode: 448
+      - type: org.osbuild.ignition
+      - type: org.osbuild.ostree.deploy
+        options:
+          osname: fedora-coreos
+          ref: localhost/fcos
+          remote: ostree-unverified-image
+          # osname: fedora-coreos
+          # ref: fedora/x86_64/coreos/stable
+          # remote: fedora
+          mounts:
+            - /boot
+            - /boot/efi
+          rootfs:
+            label: root
+          kernel_opts:
+            - rw
+            - console=tty0
+            - console=ttyS0
+            - ignition.platform.id=qemu
+            - '$ignition_firstboot'
+        inputs:
+          images:
+            type: org.osbuild.containers
+            origin: org.osbuild.source
+            mpp-resolve-images:
+              images:
+                - source: quay.io/fedora/fedora-coreos
+                  tag: stable
+                  name: localhost/fcos
+          # commits:
+          #   type: org.osbuild.ostree
+          #   origin: org.osbuild.source
+          #   references:
+          #     "2c39e9b6a6b26226b626f4daf815d5307a96b2cd57463e94f7c618e312a61980":
+          #       ref: fedora/x86_64/coreos/stable
+      - type: org.osbuild.ostree.selinux
+        options:
+          deployment:
+            osname: fedora-coreos
+            ref: localhost/fcos
+          container: true
```

Comment:
Maybe look into a better way to distinguish between ostree commit and container image inputs than having a `container` boolean?

Reply:
We don't need `container: true`. The first deployment (and we'll only ever have one deployment anyway) in ostree will always be `ref: ostree/1/1/0` so just change ref to `ref: ostree/1/1/0` here. Should work and then you can delete all the hacks you added for `--write-commit-to` and stuff.

Reply:
Using `ref: ostree/1/1/0` should work for both ostree commit and container image inputs. I just get a few warning outputs from using my actual system as the build root that mess with the output of `ostree rev-parse`, so I can't actually test the container input case (I'm using a custom `ostree` and `rpm-ostree` binary). However, by printing the output I can confirm that the parsed refspec is correct.

I can retest this once the new release of `ostree-rs-ext` is out.

Reply:
> so I can't actually test the container input case

I just tested it - seemed to work.